### DISCUSSION TITLE
missing SPAdes 4.0.0 binaries for Mac

### DISCUSSION THREAD

**Discussion by kk112024** [ablab/spades#1414] [Q&A] [answered]
Hi, where do i download SPAdes binaries for Mac from? SPAdes-4.0.0-Darwin.tar.gz seems to be missing from the release.

**Comment by asl** [comment]
It is right here, in the release: https://github.com/ablab/spades/releases/tag/v4.0.0

**Reply by kk112024** [reply]
There are multiple files but SPAdes-4.0.0-Darwin.tar.gz is not there. 
<img width="934" alt="Screenshot 2024-11-19 at 19 34 59" src="https://github.com/user-attachments/assets/8c2230c8-f94b-4ef7-8add-d24458870f04">

**Comment by asl** [comment, marked as answer]
Sure. As written in the release notes:
> Native support for running on Apple Silicon processors;

So, you need to chose depending on the CPU / architecture you're going to run SPAdes on.

**Comment by kk112024** [comment]
OK, thank you. It wasn't overly clear for a novice. But I now I need to use SPAdes-4.0.0-Darwin-arm64.tar.gz
